string                     pose_name
---
geometry_msgs/PoseStamped  pose
